### ISSUE TITLE
When using the terminal there is a huge lag up to 30 seconds

### ISSUE THREAD

**Issue by LenReinhart** [sipeed/LicheePi4A#11]
When I first got the board it was working fine. I am configured with a HDMI monitor and wireless keyboard and mouse, power is through the barrel jack. I then did and update and upgrade, added ssh and the build environment. Since then there has been this huge lag when using the terminal. I saw in the Wiki that there is a double buffering problem. It was running fine before I installed the software, How do I stop this?

**Comment by abelletti** [comment]
That was exactly my experience with the pre-loaded image. Flashing the latest image completely eliminated the issue. Have you tried that?

**Comment by LenReinhart** [comment]
No, silly me, I thought upgrading the software would get me the latest. I guess not. Thanks.
